app/uk/gov/hmrc/exports/services/WCOPointerMappingService.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services
+
+import javax.inject.Inject
+import uk.gov.hmrc.exports.models.{Pointer, PointerMapping, PointerPattern}
+import uk.gov.hmrc.exports.util.FileReader
+
+class WCOPointerMappingService @Inject()(fileReader: FileReader) {
+
+  private lazy val mappings: Set[PointerMapping] = {
```

Comment:
Very good! 
I would have suggested a Json structure as it helps understand what are we mapping more clearly and it also allows for better flexibility and allows for better deserialisation support, i.e, imagine if we want to provide error messages or other information, probably better than having to splic by comma. Probably a YAGNI at this point.

Reply:
Good suggestion, it'd be easy to refactor in as we could just write a Json Formatter for `PointerMapping`

For the purpose of this story though I think its okay for now

Reply:
Not withstanding the possibility of moving to Json - did Patryk not introduce some CSV helper package recently?